yolox/utils/logger.py

```diff
@@ -438,3 +438,329 @@ def initialize_wandb_logger(cls, args, exp, val_dataset):
+        - **YOLOX_MLFLOW_FLATTEN_PARAMS** (`str`, *optional*, defaults to `False`):
+            Whether to flatten the parameters dictionary before logging.
+        - **MLFLOW_TRACKING_URI** (`str`, *optional*):
+            Whether to store runs at a specific path or remote server. Unset by default, which skips setting the
+            tracking URI entirely.
+        - **MLFLOW_EXPERIMENT_NAME** (`str`, *optional*, defaults to `None`):
+            Whether to use an MLflow experiment_name under which to launch the run. Default to `None` which will point
+            to the `Default` experiment in MLflow. Otherwise, it is a case-sensitive name of the experiment to be
+            activated. If an experiment with this name does not exist, a new experiment with this name is created.
+        - **MLFLOW_TAGS** (`str`, *optional*):
+            A string dump of a dictionary of key/value pair to be added to the MLflow run as tags. Example:
+            `os.environ['MLFLOW_TAGS']='{"release.candidate": "RC1", "release.version": "2.2.0"}'`.
+        - **MLFLOW_NESTED_RUN** (`str`, *optional*):
+            Whether to use MLflow nested runs. If set to `True` or *1*, will create a nested run inside the current
+            run.
+        - **MLFLOW_RUN_ID** (`str`, *optional*):
+            Allow to reattach to an existing run which can be useful when resuming training from a checkpoint. When
+            `MLFLOW_RUN_ID` environment variable is set, `start_run` attempts to resume a run with the specified run ID
+            and other parameters are ignored.
+        - Other MLflow environment variables: For changing default logging Behaviour refer mlflow environment variables:
+        https://mlflow.org/docs/latest/python_api/mlflow.environment_variables.html
+        - Setup ``Databricks`` integration with MLflow: Provide these two environment variables:
+            DATABRICKS_HOST="https://adb-4273978218682429.9.azuredatabricks.net"
+            DATABRICKS_TOKEN="dapixxxxxxxxxxxxx"
+        """
+        args = trainer.args
+        self._tracking_uri = os.getenv("MLFLOW_TRACKING_URI", None)
+        self._experiment_name = os.getenv("MLFLOW_EXPERIMENT_NAME", None)
+
+        self._mlflow_log_artifacts = os.getenv("YOLOX_MLFLOW_LOG_MODEL_ARTIFACTS", "False").upper() in ENV_VARS_TRUE_VALUES
+        self._mlflow_log_model_per_n_epochs = int(os.getenv("YOLOX_MLFLOW_LOG_MODEL_PER_n_EPOCHS", 30))
+        self._mlflow_log_nth_epoch_models = os.getenv("YOLOX_MLFLOW_LOG_Nth_EPOCH_MODELS", "False").upper() in ENV_VARS_TRUE_VALUES
+        self.run_name = os.getenv("YOLOX_MLFLOW_RUN_NAME", None)
+        self.run_name = None if len(self.run_name.strip())==0 else self.run_name
+        self._flatten_params = os.getenv("YOLOX_MLFLOW_FLATTEN_PARAMS", "FALSE").upper() in ENV_VARS_TRUE_VALUES
+
+        self._nested_run = os.getenv("MLFLOW_NESTED_RUN", "FALSE").upper() in ENV_VARS_TRUE_VALUES
+        self._run_id = os.getenv("MLFLOW_RUN_ID", None)
+
```

Comment:
plz delete this empty line here. Two lines of empty in a function is not a good coding style.